repos/spack_repo/builtin/packages/py_torchgeo/package.py

```diff
@@ -240,6 +242,8 @@ class PyTorchgeo(PythonPackage):
     conflicts("py-kornia@0.6.10:", when="@:0.4.0")
     # https://github.com/microsoft/torchgeo/pull/2484
     conflicts("py-lightning@=2.5.0")
+    # https://github.com/torchgeo/torchgeo/pull/3052
+    conflicts("py-lightning@=2.5.5")
```

Comment:
```suggestion
    conflicts("py-lightning@2.5.5:2.5")
```
2.5.6 is also broken, but this will be fixed in 2.6.0 this week.

Let's also move this above the previous block so all lightning conflicts are sorted from newest to oldest, just like the versions.